### ISSUE TITLE
Set up automation for "signal_detection_status_log" from Knack to OpenData Portal

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#7018]
Allyson
MMC 

- They need the detector events to create the dashboard to help Brian VDW's maintenance reporting requirements

**Comment by dianamartin** [comment]
@amenity @johnclary please prioritize in Dev Planning

**Comment by dianamartin** [comment]
John said to add the API View fields first and then put back in "Backlog"

**Comment by amenity** [comment]
@dianamartin - just let @chiaberry know when the Knack view is set up

**Comment by chiaberry** [comment]
@dianamartin I merged the changes this morning, the dataset is scheduled to update everyday at 5a.

**Comment by dianamartin** [comment]
thanks! @chiaberry I'll check it tomorrow

**Comment by dianamartin** [comment]
Checked dataset and looks to have updated for 8/31. 

Emailed Lance & Allyson
>Hi Allyson/Lance, 
 - I’ve been able to add the Signal Detection Status Log to the OpenData portal. Right now it’s published in a “private” status, I’d like for y’all to look at it before I published it “Publicly”. 
- Lance, I tried to share it with you, but I don’t think you have an individual account created in Socrata.
- Let me know if the dataset is good to go and I’ll publish it to the world!
